routes/index.js

```diff
@@ -47,4 +47,18 @@ router.post('/registration/:badge', jsonParser, function(req, res) {
   res.redirect('/badgein');
 });
 
+router.get('/userManagement', function(req, res) {
+  console.log(req.body)
+  
+
+  var jsonParser = dbAPI.getUsers('2000-01-01', '3000-01-01').done(function(ret){
```

Comment:
I don't think you need to catch the return value here. Also instead of `.done()` please use `.then()`